### PR TITLE
Send message by locale

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

File: app/jobs/send_message_job.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@ def perform(recipient_id)
       return
     end
 
-    MessageService.new.send_message(recipient, recipient.message_batch.message_template.body)
+    message_body = recipient.message_batch.message_template.body_backend.read(recipient.preferred_language)
```
Comment (mrotondo) [inline app/jobs/send_message_job.rb:12]: No strong preference, but would `Mobility.with_locale(recipient.preferred_language) { recipient.message_batch.message_template.body }` work here? Feels slightly more clear to me.

File: spec/factories/message_templates.rb [hunk 2]
```diff
@@ -15,6 +15,11 @@
 FactoryBot.define do
   factory :message_template do
     name { 'SimpleMessage' }
-    body { 'Simple message body' }
+    # body { 'Simple message body' }
```
Comment (mrotondo) [inline spec/factories/message_templates.rb:18]: Did you intend to delete this?

File: spec/jobs/send_message_job_spec.rb [hunk 3]
```diff
@@ -2,24 +2,49 @@
...
+          allow(MessageService).to receive(:new) { message_service }
+          Mobility.with_locale(preferred_language) { expect(message_service).to receive(:send_message).with(recipient, recipient.message_batch.message_template.body) }
+          described_class.perform_now recipient.id
+        end
+      end
+
+      context "when the recipient's language preference is Spanish" do
+        let (:preferred_language) { :es }
+
+        it "sends a message in English" do
```
Comment (mrotondo) [inline spec/jobs/send_message_job_spec.rb:34]: Should this say "sends a message in Spanish"?

**Review by mrotondo — APPROVED** [review]

File: app/jobs/send_message_job.rb (hunk 1 above)
Comment (vrajmohan) [reply app/jobs/send_message_job.rb:12]: I was debating between the 2 and wondering whether an explicit `set_body(locale)` method on `MessageTemplate` would be better.

**Pull request merged**